### ISSUE TITLE
Jabber Link Corrupted

### ISSUE THREAD

**Issue by casey** [Enegnei/This-Month-In-Bitcoin-Privacy#19]
Thanks for TMIB! It's a great way to keep up with developments in the space.

I noticed in the latest issue that the jabber link is corrupted, the URL is `https://enegnei.github.io/This-Month-In-Bitcoin-Privacy/March_2021/j9roem@xmpp.jp`, and I assume it should just be `j9roem@xmpp.jp`.

**Comment by Enegnei** [comment]
Yeah I'm aware of this, it has to do with [config.yml](https://github.com/Enegnei/This-Month-In-Bitcoin-Privacy/blob/master/_config.yml) not seeming to recognise the Jabber ID. For the email address and Matrix ID, it formats them properly even though they are not typical URLs. Not sure how to fix it.
